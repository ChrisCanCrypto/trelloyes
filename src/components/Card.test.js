import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';

describe('Card component', () => {
	// Smoke test
	it('renders without crashing', () => {
		const div = document.createElement('div');
		ReactDOM.render(<Card />, div);
		ReactDOM.unmountComponentAtNode(div);
	});

	// Snapshot Test
	it('renders the the Card as expected', () => {
		const tree = renderer.create(<Card title="Title" content="content" />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
