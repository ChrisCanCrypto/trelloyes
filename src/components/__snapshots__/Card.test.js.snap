// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component renders the the Card as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    Title
  </h3>
  <p>
    content
  </p>
</div>
`;

exports[`renders the UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    Title
  </h3>
  <p>
    content
  </p>
</div>
`;
